config-controller/Makefile

```diff
@@ -152,50 +152,19 @@ undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.
 	$(KUSTOMIZE) build config/default | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
 ##@ Dependencies
-
-## Location to install dependencies to
-LOCALBIN ?= $(shell pwd)/bin
-$(LOCALBIN):
-	mkdir -p $(LOCALBIN)
+PROJECT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+include $(PROJECT_DIR)/../make/gotools.mk
 
 ## Tool Binaries
-KUBECTL ?= kubectl
```

Comment:
Should we replace KUBECTL commands with retry-kubectl.sh similar to the operator's makefile?

Reply:
Good catch I should not remove this line. I think it's fine. It's not a part of CI so maybe we don't need to auto retry